### PR TITLE
docs: fix typos

### PR DESCRIPTION
**Pull request by omahs**
docs: fix typos

### CODE REVIEW

**Review by kiner-tang — APPROVED** [review]

**Comment by thinkasany** [comment]
Can you help us modify the zh-cn doc?
image

**Comment by omahs** [comment]
@thinkasany Hey, sure! Just added the changes

File: docs/course/contract-init.zh-CN.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ order: 1
 `Solidity Compiler` 是合约编译器界面，界面中默认展示编译器的基础配置项和`Advanced Configurations` 按钮打开高级配置面板。  
 `Deploy&Run` 是为了将交易发送到当前的`环境`中。  
 `Debugger` 是一个调试器，调试交易的时候，调试器会显示合约的状态。  
-`Plugin mananer`是插件管理器，里边有非常多的插件可以选择安装。  
+`Plugin manager`是插件管理器，里边有非常多的插件可以选择安装。
```
Comment (thinkasany) [inline docs/course/contract-init.zh-CN.md:43]:
```suggestion
`Plugin Manager`是插件管理器，里边有非常多的插件可以选择安装。  
```

File: docs/course/contract-init.md [hunk 2]
```diff
@@ -33,7 +33,7 @@ As you can see, `Remix` consists of three panels and a terminal.
...
+A brief introduction to the side panel icon function, we will have a more detailed introduction when we use it later. `Home` can always open the main page, even if it is turned off. `File explorer` is used to manage workspaces and files. `Search` is a global search function. `Solidity Compiler` is the interface of the contract compiler. The interface displays the basic configuration items of the compiler by default, and the `Advanced Configurations` button opens the advanced configuration panel. `Deploy&Run` is to send the transaction to the current `environment`. `Debugger` is a debugger. When debugging a transaction, the debugger will display the status of the contract. `Plugin manager` is a plug-in manager, there are many plug-ins to choose from. `Setting` will have some basic settings, such as `language`, `theme`, `GitHub access token`, `general settings`, etc.
```
Comment (thinkasany) [inline docs/course/contract-init.md:36]:
```suggestion
A brief introduction to the side panel icon function, we will have a more detailed introduction when we use it later. `Home` can always open the main page, even if it is turned off. `File explorer` is used to manage workspaces and files. `Search` is a global search function. `Solidity Compiler` is the interface of the contract compiler. The interface displays the basic configuration items of the compiler by default, and the `Advanced Configurations` button opens the advanced configuration panel. `Deploy&Run` is to send the transaction to the current `environment`. `Debugger` is a debugger. When debugging a transaction, the debugger will display the status of the contract. `Plugin Manager` is a plug-in manager, there are many plug-ins to choose from. `Setting` will have some basic settings, such as `language`, `theme`, `GitHub access token`, `general settings`, etc.
```

**Review by thinkasany — APPROVED** [review]

**Comment by thinkasany** [comment]
> @thinkasany Hey, sure! Just added the changes

Thanks for your contribution!

**Pull request merged**